frontend/test/metabase/scenarios/filters/view.cy.spec.js

```diff
@@ -37,15 +37,15 @@ describe("scenarios > question > view", () => {
               "display-name": "CATEGORY",
               type: "dimension",
               dimension: ["field", PRODUCTS.CATEGORY, null],
-              "widget-type": "id",
+              "widget-type": "string/=",
```

Comment:
@nemanjaglumac Can you help take a look that this change is fine? I need to correct the widget type to match the actual field type so `<FieldValuesWidget />` show the correct widget type.

The [recent E2E test failure](https://www.deploysentinel.com/ci/runs/63a32c02a622c2ef12225f15) is because we mock the widget type wrong.

Since both **product.category** and **product.vendor** are both string, when we try to map the parameter using the UI, the type will be defaulted to `string/=`
<img width="330" alt="image" src="https://user-images.githubusercontent.com/1937582/209128119-003262fe-6c8d-47b6-b896-898658562a6c.png">

And with this new change that takes parameter type into account, we need to make sure we mock the correct parameter type.

Reply:
I don't remember exactly what was going on in this test, but your change makes sense.

Thank you